cdm/core/src/main/java/ucar/nc2/Attribute.java

```diff
@@ -766,6 +763,17 @@ public Builder setEnumType(EnumTypedef enumtype) {
       return this;
     }
 
+    public Builder setNumericValue(Number val, boolean isUnsigned) {
```

Comment:
The signature is pretty obvious though...so maybe not a big deal? I'd like to get to a point once the public api is well defined that we have doc strings for everything.

Reply:
The API is still changing, mostly, but not all tweaks.
Im inclined to wait until things are mostly done before adding the javadoc. 
I think it would be good to generate the "public api" javadoc, and then make that as useable as possible.
But maybe Ill get started on that sooner than later...
There are some "javadoc exemptions" according to google style guide (https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required). Mostly setters and getters. This setNumericValue() does deserve javadoc, though.

Reply:
I think waiting is ok. I just wanted to get a handle on where things stood.